test/Tests/src/Data/Locale_Spec.enso

```diff
@@ -74,4 +76,7 @@ spec = Test.group "Locale" <|
         Locale.uk . should_equal Locale.uk
         Locale.uk . should_not_equal Locale.us
 
+    Test.specify "Should correctly auto-discover static Locales" <|
+        Locale.widget_options . should_equal [(Option 'Bangladesh' 'Locale.bangladesh' [] ''), (Option 'Brazil' 'Locale.brazil' [] ''), (Option 'Canada English' 'Locale.canada_english' [] ''), (Option 'Canada French' 'Locale.canada_french' [] ''), (Option 'China' 'Locale.china' [] ''), (Option 'France' 'Locale.france' [] ''), (Option 'Germany' 'Locale.germany' [] ''), (Option 'India English' 'Locale.india_english' [] ''), (Option 'India Hindi' 'Locale.india_hindi' [] ''), (Option 'Indonesia' 'Locale.indonesia' [] ''), (Option 'Italy' 'Locale.italy' [] ''), (Option 'Japan' 'Locale.japan' [] ''), (Option 'Mexico' 'Locale.mexico' [] ''), (Option 'Nigeria' 'Locale.nigeria' [] ''), (Option 'Pakistan English' 'Locale.pakistan_english' [] ''), (Option 'Pakistan Urdu' 'Locale.pakistan_urdu' [] ''), (Option 'Poland' 'Locale.poland' [] ''), (Option 'Russia' 'Locale.russia' [] ''), (Option 'South Korea' 'Locale.south_korea' [] ''), (Option 'UK' 'Locale.uk' [] ''), (Option 'US' 'Locale.us' [] '')]
```

Comment:
Won't it be simpler to check only the locale names, not whole options?

Reply:
Done -- I'm checking the locale display names now. Is that what you meant?

Reply:
Yep. Looks good.

Now just need to verify that it displays as expected in the GUI.